### ISSUE TITLE
Invisible, unbreakable blocks.

### ISSUE THREAD

**Issue by JustaFloatingEyeYT** [CoolMineman/BitsAndChisels#107]
On a 1.18.1 server, whenever I chisel a block it will turn the block invisible and act as a barrier block (cannot be broken).

**Comment by CoolMineman** [comment]
Are you using sodium without indium?

**Comment by JustaFloatingEyeYT** [comment]
Yep I was, someone told me to get Indium when I was fixing an issue for another mod, I was gonna test if bits and chisels now works with Indium.
